R/cols_align.R

```diff
@@ -33,13 +33,20 @@
 #' allow **gt** to automatically choose the alignment of each column based on
 #' the data type (with the `auto` option).
 #'
-#' @param data *The gt table data object*
+#' @param data *The gt table or gt group data object*
```

Comment:
a LOT of the other cols_ functions have @InheritParams cols_align so I can't change this without changing all of them really